src/Symfony/Component/Notifier/Transport/AbstractTransport.php

```diff
@@ -80,7 +80,7 @@ public function send(MessageInterface $message): void
 
     abstract protected function doSend(MessageInterface $message): void;
 
-    protected function getEndpoint(): ?string
+    protected function getEndpoint(): string
```

Comment:
This is a BC break.

Existing implementations overriding this method with a `?string` return type will break because a child class is not allowed to widen the return type.

Reply:
Oh, yes of course. 

I just looked and saw this what non-public. 
By bad.

Reply:
Can we trigger a deprecation and change it in 5.3 ?